autospec/util.py

```diff
@@ -89,19 +89,52 @@ def get_sha1sum(filename):
     return sh.hexdigest()
 
 
-def print_fatal(message):
-    """Print color coded fatal errors."""
-    print("[\033[1m\033[91mFATAL\033[0m] {}".format(message))
 
 
-def print_warning(message):
-    """Print color coded warnings."""
-    print("[\033[31;1mWARNING\033[0m] {}".format(message))
+def _print_message(message, level, color=None):
+    prefix = level
+    if color:
+        # FIXME: use terminfo instead
+        if color == 'red':
+            params = '31;1'
+        elif color == 'green':
+            params = '32;1'
+        elif color == 'yellow':
+            params = '33;1'
+        elif color == 'blue':
+            params = '34;1'
+        prefix = f'\033[{params}m{level}\033[0m'
+    print(f'[{prefix}] {message}')
 
 
-def print_infile(message):
-    """Print colored INIFILE content."""
-    print("[\033[1;34mINFILE\033[0m] {}".format(message))
+def print_error(message, level='ERROR'):
```

Comment:
It seems odd to me these functions would take a level parameter.

Why not just set level in the function body (or not at all and pass it as a literal like color)?

Reply:
It is a leftover from a previous iteration of my patch. I will remove the keyword parameters.

Reply:
Done